tests/lib/rules/import-scope.js

```diff
@@ -29,67 +29,85 @@ const testCases = {
         method: {
             require: [
                 "const map = require('lodash/map')",
-                "const someOtherModule = require('some-other-module')"
+                "const someOtherModule = require('some-other-module')",
+                "const fpMap = require('lodash/fp/map')"
             ],
             import: [
                 "import map from 'lodash/map'",
-                "import someOtherModule from 'some-other-module'"
+                "import someOtherModule from 'some-other-module'",
+                "import fpMap from 'lodash/fp/map'"
             ].map(fromModule)
         },
         member: {
             require: [
                 "const {map} = require('lodash')",
                 "const {map: m} = require('lodash')",
-                "const someOtherModule = require('some-other-module')"
+                "const someOtherModule = require('some-other-module')",
+                "const {map} = require('lodash/fp')"
             ],
             import: [
                 "import {map} from 'lodash'",
                 "import {map as m} from 'lodash'",
-                "import someOtherModule from 'some-other-module'"
+                "import someOtherModule from 'some-other-module'",
+                "import {map} from 'lodash/fp'",
+                "import {map as fpMap} from 'lodash/fp'"
             ].map(fromModule)
         },
         full: {
             require: [
                 "const _ = require('lodash')",
-                "const {someMember} = require('some-other-module')"
+                "const {someMember} = require('some-other-module')",
+                "const fp = require('lodash/fp')"
             ],
             import: [
                 "import _ from 'lodash'",
                 "import * as _ from 'lodash'",
-                "import {someMember} from 'some-other-module'"
+                "import {someMember} from 'some-other-module'",
+                "import fp from 'lodash/fp'"
             ].map(fromModule)
         }
     },
     invalid: {
```

Comment:
You're right.
This plugin doesn't support `lodash/fp`, so this rule shouldn't warn against these cases. If anything, there should be tests with `lodash/fp` in the *valid* section, to make sure the rule doesn't warn against them.